scripts/get-data.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
Was this supposed to be part of the PR?

Reply:
hmm... sort of in that I needed to make the change to make the script work on my machine and thought it might be portable -- it didn't work as is on my nixos machine, though that might be my problem. I have a whole stack of changes like this that I was going to save for another PR, but this one was needed to get the tests working.

Reply:
happy to remove it if you don't like it and/or make it part of another PR that uses /usr/bin/env for all the Rscript, python, and bash

Reply:
I was just curious really.   There are a lot of shebang bash scripts and /usr/bin/env may be more correct for this type of scripting job.  If it passes the CI then its fine with me.